src/honeybot/plugins/downloaded/emoji/main.py

```diff
@@ -0,0 +1,56 @@
+class Plugin:
+    def __init__(self):
+        pass
+
+    def __meaning(emoji):
+        d = {
+            "😀": "Grinning Face",
+            "😃": "Grinning Face with Big Eyes",
+            "😄": "Grinning Face with Smiling Eyes",
+            "😁": "Beaming Face with Smiling Eyes",
+            "😆": "Grinning Face with Sweat",
+            "🤣": "Rolling on the Floor Laughing",
+            "😂": "Face with Tears of Joy",
+            "🙂": "Slightly Smiling Face",
+            "🙃": "Upside-Down Face",
+            "😉": "Winking Face",
+            "😊": "Smiling Face with Smiling Eyes",
+            "😇": "Smiling Face with Halo",
+            "🥰": "Smiling Face with Hearts",
+            "😍": "Smiling Face with Heart-Eyes",
+            "🤩": "Star-Struck",
+            "😘": "Face Blowing a Kiss",
+            "😗": "Kissing Face",
+            "😉": "Winking Face",
+            "😊": "Smiling Face with Smiling Eyes",
+            "😇": "Smiling Face with Halo",
+            "🥰": "Smiling Face with Hearts",
+            "😍": "Smiling Face with Heart-Eyes",
+            "🤩": "Star-Struck",
+            "😘": "Face Blowing a Kiss",
+            "😗": "Kissing Face",
+            "😚": "Kissing Face with Closed Eyes",
+            "😙": "Kissing Face with Smiling Eyes",
+            "😋": "Face Savoring Food",
+            "😛": "Face with Tongue",
+            "😜": "Winking Face with Tongue",
+            "🤪": "Zany Face",
+            "😝": "Squinting Face with Tongue",
+            "🤑": "Money-Mouth Face",
+            "🤗": "Smiling Face with Open Hands",
+            "🤭": "Face with Hand Over Mouth",
+        }
```

Comment:
This is good but could we perhaps use a package or a web api instead to make it future proof and easier to maintain?

Example: 
https://carpedm20.github.io/emoji/docs/#extracting-emoji
https://developer.emojidex.com/

Reply:
Alright. I'll do that

Reply:
I've deleled the log folder